geniza/corpus/views.py

```diff
@@ -137,6 +137,7 @@ def get_context_data(self, **kwargs):
             {
                 "page_title": self.page_title(),
                 "page_description": self.page_description(),
+                "id": "detail",
```

Comment:
Let's specify document or document-detail, since we could have other kinds of detail pages.

And now that I see it again I'm not sure about using `id` for the attribute here, that seems potentially misleading. Could you come up with something a bit more specific to how this is used?

Reply:
I ended up going with `page_type` here, let me know if that doesn't make sense!